torchgeo/datasets/loveda.py

```diff
@@ -0,0 +1,302 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+"""LoveDA dataset."""
+
+import glob
+import os
+from typing import Callable, Dict, List, Optional
+
+import matplotlib.pyplot as plt
+import numpy as np
+import torch
+from matplotlib.figure import Figure
+from PIL import Image
+from torch import Tensor
+
+from .geo import VisionDataset
+from .utils import download_and_extract_archive
+
+
+class LoveDA(VisionDataset):
+    """LoveDA dataset.
+
+    The LoveDA <https://github.com/Junjue-Wang/LoveDA> datataset is a
+    semantic segmentation dataset.
+
+    Dataset features:
+
+    * 2713 urban scene and 3274 rural scene HSR images with a total
```

Comment:
Any idea what the image source is? Which satellite/plane/drone?

Reply:
The authors state that they got all imagery from Google Earth so depending on the resolution, that would either be satellite or plane I suppose.

Reply:
Google Earth gets its images from difference sources such as Landsat, Copernicus, Maxar etc so its hard to say. I think a significant portion of their imagery is from Maxar. If that's the case I would guess it's Worldview 3 (0.3 m resolution).